### PR TITLE
#266 renaming UM to U-M text

### LINKED ISSUE #266: UM should be U-M
UM should be U-M to stay consistent with [University Branding (Editorial Resources).](https://brand.umich.edu/editorial-resources/abbreviations/)

Janel Ilar  11:51 AM
Update on "Add Non-UM Users", it can remain as "Add Non-UM Users".

### PR DESCRIPTION
**Pull request by pushyamig**
Fixes #266 

See the Github pages changes from here https://pushyamig.github.io/canvas-course-manager-next/

### CODE REVIEW

File: docs/add-non-um-users.html [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
 layout: default
-title: Add Non-UM Users
+title: Add Non&#8209;UM Users
 ---
 
 <div id="content-wrap">
-  <h1>Add Non-UM Users Documentation</h1>
+  <h1>Add Non&#8209;UM Users Documentation</h1>
```
Comment (ssciolla) [inline docs/add-non-um-users.html:7]: Maybe I'm missing something, but the non-breaking special character doesn't seem to be rendered properly in the documentation (the navigation links look okay though).
Screen Shot 2021-12-13 at 12 33 17 PM
.

**Review by ssciolla — COMMENTED** [review]
See comment below. It doesn't seem essential we use this special character (or at least not in all cases), but hopefully we can get it to work.

File: docs/add-non-um-users.html (hunk 1 above)
Comment (pushyamig) [reply docs/add-non-um-users.html:7]: Hmm... I am not seeing this with browsers(chrome, safari, edge, FF) i tested. I really don't want to complicate this. I chose to go back to regular (`-`)
Screen Shot 2021-12-14 at 8 45 25 AM

Comment (pushyamig) [reply docs/add-non-um-users.html]: @ssciolla I made some [change](https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/268/commits/a18d450d286755ec33fc60918e7a0c0782a80eeb). See if this is any better. I agree not all needs to be encoded.

File: docs/_data/navigation.yml [hunk 2]
```diff
@@ -4,11 +4,11 @@
   link: /merge-sections.html
 - name: Create Sections
   link: /create-sections.html
-- name: Canvas Gradebook Formatter
+- name: Format Gradebook Formatter
```
Comment (ssciolla) [inline docs/_data/navigation.yml:7]:
```suggestion
- name: Format Canvas Gradebook
```

File: docs/gradebook-thirdparty.html [hunk 3]
```diff
@@ -1,9 +1,9 @@
 ---
 layout: default
-title: Gradebook Converter Third-Party
+title: Format Third-Party Gradebook
 ---
 
 <div id="content-wrap">
-  <h1>Gradebook Converter Third-Party Documentation</h1>
+  <h1>Format Third-Party Documentation</h1>
```
Comment (ssciolla) [inline docs/gradebook-thirdparty.html:7]:
```suggestion
  <h1>Format Third-Party Gradebook Documentation</h1>
```

File: docs/add-um-users.html [hunk 4]
```diff
@@ -1,9 +1,9 @@
...
 layout: default
-title: Add UM Users
+title: Add U-M Users
 ---
 
 <div id="content-wrap">
-  <h1>Add UM Users Documentation</h1>
+  <h1>Add U-M Users Documentation</h1>
   <p>Follow documentation for this Features</p>
 </div>
```
Comment (ssciolla) [inline docs/add-um-users.html:9]:
```suggestion
</div>

```

File: docs/gradebook-canvas.html [hunk 5]
```diff
@@ -1,9 +1,9 @@
...
 layout: default
-title: Gradebook Converter
+title: Format Canvas Gradebook
 ---
 
 <div id="content-wrap">
-  <h1>Gradebook Converter Documentation</h1>
+  <h1>Format Canvas Gradebook Documentation</h1>
   <p>Follow documentation for this Features</p>
 </div>
```
Comment (ssciolla) [inline docs/gradebook-canvas.html:9]:
```suggestion
</div>

```

File: docs/gradebook-thirdparty.html [hunk 6]
```diff
@@ -1,9 +1,9 @@
...
   <p>Follow documentation for this Features</p>
 </div>
```
Comment (ssciolla) [inline docs/gradebook-thirdparty.html:9]:
```suggestion
</div>

```

**Review by ssciolla — APPROVED** [review]
I'm approving, but I added a handful of suggestions/comments that I'd like considered. No need to request another review.

File: ccm_web/client/src/models/feature.ts [hunk 7]
```diff
@@ -29,7 +29,7 @@ const formatCanvasGradebookProps: FeatureDataProps = {
 
 const formatThirdPartyGradebookProps: FeatureDataProps = {
   id: 'FormatThirdPartyGradebook',
-  title: 'Format Third-Party Gradebook',
+  title: 'Format Third\u2011Party Gradebook',
   description: 'Format a CSV file exported from an external tool for importing grades into the Canvas Gradebook.',
```
[the remark this answers is not in the document]
Comment (pushyamig) [reply ccm_web/client/src/models/feature.ts]: Done

**Comment by pushyamig** [comment]
@ssciolla I resolved all the comments and suggestion. So I am merging it.

**Pull request merged**